### PR TITLE
visit_struct: add recipe

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **visit_struct**

visit_struct is used by structopt internally.
I want to separate it.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — DISMISSED** [review]
LGTM

File: recipes/visit_struct/all/conandata.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+sources:
+  "1.0":
+    url: "https://github.com/garbageslam/visit_struct/archive/refs/tags/v1.0.tar.gz"
+    sha256: "7e5d727bd9fbe444d182840625f5eb21f2cf67f06f3aeab27a2d7c4724b3c09c"
+
```
Comment (jgsogo) [inline recipes/visit_struct/all/conandata.yml:5]:
```suggestion
    sha256: "7e5d727bd9fbe444d182840625f5eb21f2cf67f06f3aeab27a2d7c4724b3c09c"
```

**Review by jgsogo — DISMISSED** [review]
Suggesting to fix the linter warning, but approving. Let's see what happens first 😄

**Review by jgsogo — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
LGTM

**Review by SSE4 — APPROVED** [review]

**Pull request merged**